modules/sdk/gradle-plugins-workspace/src/main/resources/schemas/client-extension.schema.json

```diff
@@ -217,6 +217,14 @@
 					"allOf": [
 						{
 							"$ref": "#/definitions/propertySets/properties/url"
+						},
+						{
+							"properties": {
+								"scriptElementAttributes": {
+									"type": "object"
```

Comment:
Great, added title and description based on existing usages in `client-extension.schema.json`

Reply:
Oh that's great. So I proposed:

```
Set a list of attributes that will be added to the script appended to the page. Do not specify the 'src' attribute; instead, use the 'url' property. Boolean attributes must have their values explicitly declared.
```

thoughts @ethib137 ?

Reply:
"A list of HTML attributes that will be added to the script element."

Do we need more than this? Adding the extra details seems a bit overcomplicated here. Wouldn't we have validation to handle those other pieces. @drewbrokke What do you think?

Reply:
@ethib137  You did not ask me, but I'll also comment on this :sweat_smile: 

Preventing errors is better than handling them. Having the validation is essential, so is informing the user what **not to do** before they do something

Reply:
So, if we keep that info on the description:

```
A list of HTML attributes that will be added to the script element. Do not specify the 'src' attribute; instead, use the 'url' property.
```

Boolean attributes will probably be declared with explicitly values since that's the way of YAML works. So we can shorter the message

Reply:
@thiago-buarqque another option we have here is to actually build the "no `src` attribute" requirement into the schema. You can add negative requirements. That way it won't have to be explicitly explained in the description.

Reply:
See https://json-schema.org/understanding-json-schema/reference/combining#not

Reply:
Oh, that's a clever solution. Missed that 🤦‍♂️

@drewbrokke  For instance:

```
...,
"not": { 
    "properties": {"src": {}}, 
    "additionalProperties": false 
} 
```

and the description would just be

```
A list of HTML attributes that will be added to the script element.
```

Reply:
Sent the changes I described on the comment above.

@ethib137, tomorrow (wednesday) is a holiday in my province (state). If everything is alright with the changes I proposed, please, forward the PR. Thank you :)

Reply:
@thiago-buarqque I'm not sure if that's a valid schema. You may need to add an `allOf` statement to basically say "this schema matches all of these schemas".

Reply:
I checked on https://www.jsonschemavalidator.net/, it seems to work fine. Maybe I'm missing something?

Reply:
Does it effectively validate the `yaml` files?

Reply:
I see what you mean. Gonna try it. I assumed it did. My mistake

Reply:
You should be able to apply local schema files to your yaml file for testing with a comment: [using-inlined-schema](https://github.com/redhat-developer/yaml-language-server#using-inlined-schema)

Reply:
This looks good! Thanks guys!

Reply:
By doing a manual test, the error message is not the best. Is that something we're able to handle some way? @drewbrokke 
![image](https://github.com/liferay-platform-experience/liferay-portal/assets/57292581/6dc22fab-2ef9-4f18-8a5b-b840a4a9cd59)

Reply:
Hey @thiago-buarqque can you try this other approach to see if it generates a clearer error message?
```
"scriptElementAttributes": {
	"additionalProperties": {
		"type": [
			"boolean",
			"integer",
			"number",
			"string"
		]
	},
	"description": "A list of HTML attributes that will be added to the script element.",
	"properties": {
		"src": {"not": {}}  /* not allowed to be anything */
	},
	"title": "Script Element Attributes",
	"type": "object"
}
```

Reply:
Also [apparently this can be simplified](https://stackoverflow.com/questions/30515253/json-schema-valid-if-object-does-not-contain-a-particular-property) to:
```
{
    "properties": {
       "src": false
    }
}
```

Reply:
Maybe try this as well:
```
  "properties": {
    "src": { "not": {}, "errorMessage": "Use the 'url' property instead of adding a 'src' attribute." }
  }
```

Reply:
@ethib137 

```
{
    "properties": {
       "src": false
    }
}
```

Give us a better error message:
![image](https://github.com/liferay-platform-experience/liferay-portal/assets/57292581/52fd6ad9-5946-41b0-b118-b83c4d64716b)

I checked the docs, it seems there is no way of providing a custom error message.
https://json-schema.org/draft/2020-12/json-schema-core#name-comments-with-comment

So `Property 'src' is not allowed` is the best we can get

Reply:
Nice! That looks good to me!